postman/schemas/index.yaml

```diff
@@ -17,11 +17,19 @@ paths:
       requestBody:
         required: true 
         content:
-          video/wav:
+          multipart/form-data:
             schema:
-              $ref: '#/components/schemas/AudioFile'
-          
-
+              type: object
+              properties:
+                AudioFile:
+                  type: object
+                  description: .wav file upload 
+                TextFile:
+                  type: object 
+                  description: .txt file upload
+                Recognizer:
+                  type: string 
+                  description: a selection of one of the speech Recognizer's for Rhubarb
       responses:
```

Comment:
I think we're missing the `characterSelect`?
In the request, we can also specify the character to animate, but we need to update the character names. We should check with the client if the official names really are Barb and "Boy"

Reply:
I mentioned this above also - I kinda think we might be able to just swap Recognizer out for the character selection, depending on what the client thinks about the differences between the two recognizers

Reply:
Instead of calling the variable `charactarSelect` we should title it `avatar` (It's less wordy and that's currently what we're calling it in our codebase!)